ui/libproject/devstart.rb

```diff
@@ -58,6 +121,15 @@ def dev_up(*args)
 
   common.run_inline %W{docker-compose run --rm --service-ports ui}
 end
+Common.register_command({
+  :invocation => "deploy-api",
```

Comment:
This should be deploy-ui

Reply:
Wasn't supposed to be there at all. :) Removed.